guides-markdown/browser.md

```diff
@@ -9,6 +9,8 @@ The bcoin JavaScript library can be easily formatted to run in a web browser lik
 Using a proxy server, we can connect the web client to the actual live Bitcoin network.
 ```
 
+> :warning: **Currently this feature is not working**
```

Comment:
hmmmm it *is* still working, just not hosted on our server any more. Maybe we can change the warning to explain that anyone can run such a service, but we aren't anymore

Reply:
![image](https://user-images.githubusercontent.com/43396684/184407809-5ccce9f3-a21f-4f24-9497-2e4c9cde7543.png)
Getting that error while following the instructions

Reply:
see https://github.com/bcoin-org/bcoin/pull/741 and https://github.com/bcoin-org/bcoin/issues/685